### PR TITLE
update: added new homepage to show available routes

### PR DESCRIPTION
**Pull request by KyeMaloy97**
Added a new home page to show what routes are available so when it goes on IBM Cloud, its obvious what you can see. Also makes the project feel more like a complete entity.

### CODE REVIEW

File: Sources/KituraSampleRouter/RouterCreator.swift [hunk 1]
```diff
@@ -88,6 +88,17 @@ public struct RouterCreator {
 
         router.all("/static", middleware: StaticFileServer())
         router.all("/chat", middleware: StaticFileServer(path: "./chat"))
+        
+        router.all("/", middleware: StaticFileServer(path: "./Views"))
+        router.get("/") { request, response, next in
+            response.headers["Content-Type"] = "text/html; charset=utf-8"
+            do {
+                try response.render("home.stencil", context: [String: Any]()).end()
```
Comment (ianpartridge) [inline Sources/KituraSampleRouter/RouterCreator.swift]: As far as I can see, this isn't a Stencil file, it's just HTML. Should it just be `home.html` and served via `response.send()`?

File: Sources/KituraSampleRouter/RouterCreator.swift [hunk 2]
```diff
@@ -163,7 +177,7 @@ public struct RouterCreator {
         }
 
         let templateEngine = StencilTemplateEngine(extension: _extension)
-        router.setDefault(templateEngine: templateEngine)
+        //router.setDefault(templateEngine: templateEngine)
```
Comment (ianpartridge) [inline Sources/KituraSampleRouter/RouterCreator.swift]: Just remove this?

File: Views/home.stencil [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<!--
```
Comment (ianpartridge) [inline Views/home.stencil]: I think older versions of Internet Explorer won't render this - they require the file to start with the `<!DOCTYPE HTML>`. I think just remove this header...

File: Views/mobile.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
+
```
Comment (ianpartridge) [inline Views/mobile.css]: We can add the licence header here though.

File: Views/stylesheet.css [hunk 5]
```diff
@@ -0,0 +1,181 @@
+
```
Comment (ianpartridge) [inline Views/stylesheet.css]: Licence here too.

File: Views/home.stencil [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+    <link rel="icon" type="image/png" href="assets/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="assets/favicon-16x16.png" sizes="16x16" />
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="stylesheet.css"/>
+    <link rel="stylesheet" type="text/css" href="mobile.css" media="(max-width: 800px)" />
+  </head>
+  <body>
+    <div class="header">
+        <div class="socialLinks">
+            <a href="http://swift-at-ibm-slack.mybluemix.net/"><img src="assets/slack.png" alt="Slack"/></a>
```
Comment (ianpartridge) [inline Views/home.stencil]: `slack.kitura.io`

**Comment by KyeMaloy97** [comment]
screen shot 2018-04-27 at 1 26 04 pm
screen shot 2018-04-27 at 1 25 56 pm

File: Sources/KituraSampleRouter/RouterCreator.swift (hunk 2 above)
Comment (KyeMaloy97) [reply Sources/KituraSampleRouter/RouterCreator.swift]: Done.

File: Views/home.stencil (hunk 3 above)
Comment (KyeMaloy97) [reply Views/home.stencil]: Removed

File: Views/mobile.css (hunk 4 above)
Comment (KyeMaloy97) [reply Views/mobile.css]: Done.

File: Views/stylesheet.css (hunk 5 above)
Comment (KyeMaloy97) [reply Views/stylesheet.css]: Done.

File: Views/home.stencil (hunk 6 above)
Comment (KyeMaloy97) [reply Views/home.stencil]: Fixed

File: Views/home.html [hunk 7]
```diff
@@ -0,0 +1,96 @@
+<!--
```
Comment (ianpartridge) [inline Views/home.html]: Remove this header and make the start of the file like this:

```html
<!DOCTYPE HTML>
<html>
...
```

**Pull request merged**